test-app/__tests__/webpack.config.babel.js

```diff
@@ -0,0 +1,29 @@
+import test from 'ava'
+import fs from 'fs'
+import { exec } from 'child_process'
+
+test('it exports the config', t => {
+  const originalConfig = require('../webpack.config.babel')
+  const config = Object.assign({}, originalConfig, {
+    plugins: originalConfig.plugins.map(p => p.constructor.name)
+  })
+  t.snapshot(config)
+})
+
+test.cb('it builds', t => {
+  exec('npm run build', (err, stdout, stderr) => {
+    t.is(err, null)
+
+    const html = fs.readFileSync('./build/index.html', { encoding: 'utf8' })
+    const js = fs.readFileSync('./build/bundle.js', { encoding: 'utf8' })
+    const css = fs.readFileSync('./build/css/main.0ec7a2a4.css', { encoding: 'utf8' })
```

Comment:
Doesn't the nonce in the filename change on every build?

Reply:
It will only change when something from the css changes (same for the class name). 
Ill push a new commit soon, which handles it a bit differently - thoughts welcome!